### DISCUSSION TITLE
Error Occurred : '异常'

### DISCUSSION THREAD

**Discussion by Legend94rz** [PaddlePaddle/PaddleSpatial#130] [BaiduKDDCup2022] [answered]
![image](https://user-images.githubusercontent.com/7570644/171318629-6652aec5-ef7f-4c30-98c6-e05daeae6905.png)
my team name: `ggg`

BTW, the evaluate process runs really slowly. At my local computer, the time cost of `forecast` is about several minutes, but it keeps running at least 2hours after submitting. Is there any possibility of optimization of evaluation?

**Comment by ramber1836** [comment, marked as answer]
The slow evaluation is caused by the scheduling of the online test. 
More resources will be allocated by this evening.

**Reply by Legend94rz** [reply]
what about the exception?

**Reply by ramber1836** [reply]
It is caused by the out-of-time error.

**Reply by Legend94rz** [reply]
There may also be other reason lead to the exception. I submitted again at 11:49, and if time-out after 10hours, it will error at ~ 21:00(now ~19:30). 
It shows nothing on "Error Occured" this time.
could you please check the server log?
![image](https://user-images.githubusercontent.com/7570644/171391262-f16097d9-3a69-43f3-96f3-643a19ac17f2.png)

**Reply by ramber1836** [reply]
The administrator terminated several 'zombie' jobs this afternoon and some of them have no logs. 
Another reason that may lead to the log being truncated: the buffered log is too large that the database fails to save it (the platform will be updated to fix this bug next week).
More resources have been allocated and you can resubmit your job to evaluate your code. 
Sorry for the inconvenience.
